### DISCUSSION TITLE
please for the love of god do not make it flat

### DISCUSSION THREAD

**Discussion by Tux9001** [vim/website_next_generation#76] [General]
seriously i am begging you

**Comment by hanEck** [comment]
Thank you for your participation in the project. Can you please clarify what you mean by that?

**Comment by Tux9001** [comment]
By "flat" I mean adhering to [Flat Design ](https://en.wikipedia.org/wiki/Flat_design) principles. Not only are these overdone and aesthetically unappealing to many, including myself, but they have also been scientifically proven to be less efficient to use then equivalent non-flat designs. Additionally, simply making something flat because everyone else is doing it strips it of its personality and forces it to conform to a sterile, corporate nature.
